javabuilder-authorizer/http_authorizer_function.rb

```diff
@@ -15,5 +18,45 @@ def lambda_handler(event:, context:)
 
   decoded_token = JwtHelper.decode_token(jwt_token, origin)
   return JwtHelper.generate_deny(route_arn) unless decoded_token
-  return JwtHelper.generate_allow(route_arn, decoded_token)
+
+  token_payload = decoded_token[0]
+  token_status = get_token_status(context, token_payload['sid'])
+  return JwtHelper.generate_deny(route_arn) unless token_status == TokenStatus::VALID_HTTP
+
+  JwtHelper.generate_allow(route_arn, token_payload)
+end
+
+def get_token_status(context, sid)
+  client = Aws::DynamoDB::Client.new(region: get_region(context))
+
+  begin
+    ttl = Time.now.to_i + 120
```

Comment:
nit: add a constant for 120 with a name that includes the units of that number (seconds I'm guessing?)